src/core/CCArray.mli

```diff
@@ -57,28 +69,30 @@ val length : _ t -> int
 (** [length a] returns the length (number of elements) of the given array [a]. *)
 
 val fold : ('a -> 'b -> 'a) -> 'a -> 'b t -> 'a
-(** [fold f acc a] computes [f (... (f (f acc a.(0)) a.(1)) ...) a.(n-1)],
+(** [fold f init a] computes [f (... (f (f init a.(0)) a.(1)) ...) a.(n-1)],
```

Comment:
why change this?

Reply:
bacause its called ``~init`` in ``CCArrayLabels.mli``. The whole point of this proposal is to reduce the maintanance costs of labelled interfaces, so any inconsistencies should be erased. Please note that in this case backwards compatibility is not affected. Changing the label name in ``CCArrayLabels.mli`` would break backwards compatibility.

Reply:
:+1: